index.html

```diff
@@ -1,12 +1,241 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Teardown</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="assets/css/reset.css">
 </head>
+
 <body>
-    
-</body>
+    <header class="grey-3 grid">
+        <div class="menu col-1-7">
+            <span class="logo">Logo</span>
+        </div>
+        <div class="menu col-1-7">
+            <ul>
+                <li class="menu-icon"></li>
+                <li class="menu-text">ARTICLES</li>
+                <li class="menu-subtext">Design & Development</li>
+            </ul>
+        </div>
+        <div class="menu col-1-7">
+            <ul>
+                <li class="menu-icon"></li>
+                <li class="menu-text">BOOKS</li>
+                <li class="menu-subtext">Digital & Physical Books</li>
+            </ul>
+        </div>
+        <div class="menu col-1-7">
+             <ul>
+                <li class="menu-icon"></li>
+                <li class="menu-text">EVENTS</li>
+                <li class="menu-subtext">Conferences & workshops</li>
+            </ul>
+        </div>
+        <div class="menu col-1-7">
+            <ul>
+                <li class="menu-icon"></li>
+                <li class="menu-text">JOBS</li>
+                <li class="menu-subtext">Find work & employees</li>
+            </ul>
+        </div>
+        <div class="menu col-1-7">
+            <ul>
+                <li class="menu-icon"></li>
+                <li class="menu-text">MEMBERSHIPS</li>
+                <li class="menu-subtext">Webinars & early-birds</li>
+            </ul>
+        </div>
+        <div class="menu col-1-7">
+            <button class="btn">TOPICS</button>
+
+        </div>
+    </header>
+    <div class="latest-article grid">
+        <div class="col-1-3 article grey-1">
+            djf
+        </div>
+        <div class="col-1-3 article grey-2">
```

Comment:
this div should be on top of the left and right.
Left and right div should be 50% width.